tools/configen/configen/templates/module.j2

```diff
@@ -10,7 +10,10 @@ from dataclasses import dataclass, field
 @dataclass
 class {{class_name}}Conf:
     _target_: str = "{{ classes_map[class_name].target }}"
+{% if classes_map[class_name].convert %}
+    _convert_: 'all'
+{% endif %}
```

Comment:
Any value in the default_flags dict  is copied:
loop on the default_flags dict and and add things.
key should be copied. value should be copied, and don't forget the type.

result should be.
```python
_convert_ : str = "all"
```

It should work correctly for other values of _convert_ (like "none" or "partial").